src/pages/compare/_sections/3-issues-3-companies.tsx

```diff
@@ -0,0 +1,38 @@
+import Section from "./common/section";
+import PieChartCompareCard from "../../../components/RemoteCards/PieChartCompareCard";
+import React from "react";
+
+export default function () {
+  return (
+    <Section
+      advanced
+      title='Top 50 company of Issue Openers'
```

Comment:
Maybe `Creators` is better, @septemberfd, what do you think? 

```suggestion
      title='Top 50 company of Issue Creators'
```